src/main/java/xyz/nucleoid/bedwars/util/BwPotions.java

```diff
@@ -0,0 +1,16 @@
+package xyz.nucleoid.bedwars.util;
+
+import net.minecraft.entity.effect.StatusEffectInstance;
+import net.minecraft.entity.effect.StatusEffects;
+import net.minecraft.potion.Potion;
+import net.minecraft.util.registry.Registry;
+
+public class BwPotions {
+    public static final Potion JUMP_BOOST_V = register("jump_boost_v", new Potion(new StatusEffectInstance[]{new StatusEffectInstance(StatusEffects.JUMP_BOOST, 600, 5)}));
```

Comment:
Will this function properly with vanilla clients?

Reply:
I believe so, tested from both my fabric client and vanilla client